packages/devtools/src/commands/createWindow.js

```diff
@@ -0,0 +1,12 @@
+import uuidv4 from 'uuid/v4'
+
+export default async function createWindow () {
```

Comment:
Create window doesn't work if no arguments were passed
```
Usage: createWindow(type)

Property Description:
  "type" (string): Set to 'tab' if the newly created window shares an OS-level window with the current browsing context, or 'window' otherwise.

For more info see https://w3c.github.io/webdriver/#new-window
```

I think there is something wrong with building protocol object as far as devtools command should override webdriver commands.

Also, types missing for this command, I mean the one where createWindow has no arguments AND where createWindow returns `string` but not `object`.

![image](https://user-images.githubusercontent.com/25589559/62006056-0580ec00-b13c-11e9-8fed-8bb1ba440e04.png)

Reply:
@christian-bromann thank you for changing return type from string to object.

However argument is still required for `createWindow()`

If it cannot be done right now, we can create task for it

Reply:
> However argument is still required for `createWindow()`

an argument is also required if you use WebDriver.

Reply:
Let me clarify, the argument is not needed for devtools but required for webdriver. 

Typescript types we have assumes that argument is always required. 
We need to extend this to make argument optional when using devtools.

We can ship these types along with devtools package to avoid mixing everything

Reply:
I modified the command so that it requires a value now. WebDriver and DevTools need to have the same interface.

Reply:
Can you point me to that change? I can't see arg here
`export default async function createWindow () {`

Reply:
thank you, got it!